src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json

```diff
@@ -860,7 +860,7 @@
           "Source": "Lucene",
           "Name": "RecentBlogPosts",
           "Index": "Search",
-          "Template": "[js:base64('ew0KICAicXVlcnkiOiB7DQogICAgInRlcm0iIDogeyAiQ29udGVudC5Db250ZW50SXRlbS5Db250ZW50VHlwZSIgOiAiQmxvZ1Bvc3QiIH0gDQogIH0sDQogICJzb3J0IjogeyAiQ29udGVudC5Db250ZW50SXRlbS5DcmVhdGVkVXRjIjogeyAib3JkZXIiOiAiZGVzYyIgfSB9LA0KICAic2l6ZSI6IDMNCn0=')]",
+          "Template": "[js:base64('ewogICJxdWVyeSI6IHsKICAgICJ0ZXJtIiA6IHsgIkNvbnRlbnQuQ29udGVudEl0ZW0uQ29udGVudFR5cGUiIDogIkJsb2dQb3N0IiB9IAogIH0sCiAgInNvcnQiOiB7ICJDb250ZW50LkNvbnRlbnRJdGVtLkNyZWF0ZWRVdGMiOiB7ICJvcmRlciI6ICJkZXNjIiwgImZpZWxkVHlwZSI6ICJkb3VibGUiIH0gfSwKICAic2l6ZSI6IDMKfQ==')]",
```

Comment:
Can you make it an external snippet instead? This will make it easier for the next contributor. And I assume you understand what I mean as you were able to understand this.

Reply:
After a little digging, I'm guessing this is what you mean?
410f5da4f1a93b1459965230108acb9a048860c1